eng/CodeGeneration.targets

```diff
@@ -36,21 +35,12 @@
     <Exec Command="npx autorest@$(_AutoRestVersion) --version=$(_AutoRestCoreVersion) $(AutoRestInput) $(AutoRestAdditionalParameters) --use=$(_AutoRestCSharpVersion) --output-folder=$(MSBuildProjectDirectory) --title=$(RootNamespace) --namespace=$(RootNamespace) --shared-source-folder=$(_SharedCodeDirectory)" />
   </Target>
 
+  <PropertyGroup Condition="'$(_GenerateCode)' == 'true'">
```

Comment:
What does this line do?

Reply:
Checks that we are in a generated code project.